ipvs.go

```diff
@@ -0,0 +1,199 @@
+package procfs
+
+import (
+	"bufio"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"strconv"
+	"strings"
+)
+
+type IPVSStats struct {
+	Connections, IncomingPackets, OutgoingPackets, IncomingBytes, OutgoingBytes uint64
+}
+
+type IPVSBackendStatus struct {
```

Comment:
Try running golint (https://github.com/golang/lint) - it will tell you to have docstrings for all exported types and fields :)